### PR TITLE
Adding HangFire artifact creation to azure pipeline

### PR DESCRIPTION
**Pull request by netcompany-martinropcke**
Adding artifacts for HangFire pipeline

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -28,19 +38,31 @@ steps:
...
-
+    command: publish
+    publishWebProjects: false
+    projects: '$(hangFireProject)'
+    arguments: '--configuration $(BuildConfiguration) --output $(System.DefaultWorkingDirectory)/Deploy/HangFire'
+    zipAfterPublish: true
+    modifyOutputPath: true
+  displayName: 'Build HangFire deployment package'
+  
+# this uploads API as an artifact
```
Comment (matishadow) [inline azure-pipelines.yml]: Could you start this comment with Uppercase letter i.e. `This` so it's consistent with other comments?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -2,14 +2,24 @@
 
 # execute on all branches
```
Comment (matishadow) [inline azure-pipelines.yml]: Could you start this comment with Uppercase letter i.e. `execute` so it's consistent with other comments?

File: azure-pipelines.yml [hunk 3]
```diff
@@ -28,19 +38,31 @@ steps:
...
+    publishWebProjects: false
+    projects: '$(apiProject)'
+    arguments: '--configuration $(BuildConfiguration) --output $(System.DefaultWorkingDirectory)/Deploy/API'
+    zipAfterPublish: true
+    modifyOutputPath: true
+  displayName: 'Build API deployment package'
 
-# this code takes all the files in $(Build.ArtifactStagingDirectory) and uploads them as an artifact of your build.
-- task: PublishBuildArtifacts@1
+#Create the deploy package for HangFire
```
Comment (matishadow) [inline azure-pipelines.yml]: Could you add space between `#` and `Create` so it's consistent with other comments?

**Review by matishadow — CHANGES_REQUESTED** [review]

**Review by flamaniac — APPROVED** [review]

**Review by matishadow — APPROVED** [review]

**Comment by martnc** [comment]
Updating azure pipeline to create zip files for both api and jobs.

**Pull request merged**